### ISSUE TITLE
[idea] add a finished mark along with the tutorials titles

### ISSUE THREAD

**Issue by Ayman97** [iliakan/javascript-tutorial-en#384]
Please add a mark to indicate the finished tutorials, like  :heavy_check_mark:

**Comment by iliakan** [comment]
You mean, "all read"?

**Comment by Ahmed-Magdy-S** [comment]
Yes, as @Ayman97 said, a new features like a mark will absolutely improve reading with encouraging people to finish tutorials to the end.
It's like [The Odin Project](https://www.theodinproject.com)  website if you can take a look and see what is that mean @iliakan

**Comment by Lk-ux** [comment]
It is a good suggestion.
